blitzloop/play.py

```diff
@@ -43,6 +46,9 @@
     display = graphics.Display(1280, 720, fullscreen, None)
 print(display.width, display.height)
 
+audio = AudioEngine()
```

Comment:
This was added to make sure jackd starts automatically on osx. I'll change `mpvplayer.py` to do this in https://github.com/marcan/blitzloop/pull/13.